### PR TITLE
COE-754: rename hits to frequency & clean up

### PR DESCRIPTION
**Pull request by pejo-sag**

### CODE REVIEW

File: event-based-simulators/main/cumulocityAPI.py [hunk 1]
```diff
@@ -18,15 +16,8 @@
...
 }
 
-# JSON-PYTHON mapping, to get json.load() working
-null = None
-false = False
-true = True
-######################
 log = logging.getLogger("C8yAPI")
 
-OEE_DATA_MODEL_FIELD_NAME = "@com_adamos_oee_datamodel_MachineOEEConfiguration"
```
Comment (pejo-sag) [inline event-based-simulators/main/cumulocityAPI.py:28]: Iam not sure wether this line is really needed or not, just vsc couldn't find die beeing referenced anywhere

**Comment by pejo-sag** [comment]
Ended up to be a lot of refactoring. Found some smaller issues and improved on the asset hierarchy creation

File: event-based-simulators/main/cumulocityAPI.py (hunk 1 above)
Comment (mvoigt-sag) [reply event-based-simulators/main/cumulocityAPI.py:28]: I dont think it is required

File: event-based-simulators/main/event_based_simulators.py [hunk 2]
```diff
@@ -8,21 +8,36 @@
 def current_timestamp(format = "%Y-%m-%dT%H:%M:%S.%f"):
     return datetime.utcnow().strftime(format)[:-3] + 'Z'
 
-logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.INFO)
+cumulocityAPI = CumulocityAPI()
+oeeAPI = OeeAPI()
+
+#TODO: Parameter to delete all Profiles
```
Comment (mvoigt-sag) [inline event-based-simulators/main/event_based_simulators.py:14]: that line should be obsolete now, shouldnt it?

File: event-based-simulators/main/event_based_simulators.py [hunk 3]
```diff
@@ -465,16 +458,21 @@ def load(filename):
 SHIFTPLANS_MODELS = load("shiftplans.json")
 shiftplans = list(map(lambda model: Shiftplan(model), SHIFTPLANS_MODELS))
 
-if CREATE_PROFILES.lower() == "true":
-    [oeeAPI.create_and_activate_profile(id, ProfileCreateMode.CREATE_IF_NOT_EXISTS) 
+if DELETE_PROFILES.lower() == "true":
+    log.debug(f'Deleting all Profiles')
+    oeeAPI.delete_all_simulators_profiles()
+
+if PROFILE_CREATE_MODE:
```
Comment (mvoigt-sag) [inline event-based-simulators/main/event_based_simulators.py:465]: I'm not sure I understand the syntax; does that mean that the following block is only executed if the variable 'PROFILE_CREATE_MODE' is set? But is it even possible that it is not set? b/c we have a default value defined in line 17

**Review by mvoigt-sag — APPROVED** [review]

**Review by namnhatpham — APPROVED** [review]

File: event-based-simulators/main/event_based_simulators.py (hunk 2 above)
Comment (pejo-sag) [reply event-based-simulators/main/event_based_simulators.py:14]: Done

File: event-based-simulators/main/event_based_simulators.py (hunk 3 above)
Comment (pejo-sag) [reply event-based-simulators/main/event_based_simulators.py:465]: True. This is still the syntax that i used when PROFILE_CREATE_MODE was a boolean

**Review by mvoigt-sag — APPROVED** [review]

**Pull request merged**